### PR TITLE
feat(resource): add support for several test types

### PR DESCRIPTION
**Pull request by JTBlanchard**
Add support for test types: 

- BGP ( Fixes: #1 )
- DNS Trace
- DNSSEC
- SIP Server
- RTP Stream ("voice")
- Voice Call.

### CODE REVIEW

**Comment by william20111** [comment]
Also if you could update the README checkboxes to add the coverage

File: go.mod [hunk 1]
```diff
@@ -18,6 +18,8 @@ require (
 	github.com/posener/complete v1.2.3 // indirect
 	github.com/ulikunitz/xz v0.5.7 // indirect
 	github.com/vmihailenco/msgpack v4.0.4+incompatible // indirect
-	github.com/william20111/go-thousandeyes v0.5.3
+	github.com/william20111/go-thousandeyes v0.6
 	golang.org/x/crypto v0.0.0-20200214034016-1d94cc7ab1c6 // indirect
 )
+
+replace github.com/william20111/go-thousandeyes => /Users/joshblan/code/go-thousandeyes
```
Comment (william20111) [inline go.mod:25]: will need to remove this

File: go.mod [hunk 2]
```diff
@@ -18,6 +18,8 @@ require (
 	github.com/posener/complete v1.2.3 // indirect
 	github.com/ulikunitz/xz v0.5.7 // indirect
 	github.com/vmihailenco/msgpack v4.0.4+incompatible // indirect
-	github.com/william20111/go-thousandeyes v0.5.3
+	github.com/william20111/go-thousandeyes v0.6
```
Comment (william20111) [inline go.mod:21]: v0.6.0

File: go.mod (hunk 1 above)
Comment (JTBlanchard) [reply go.mod]: Apologies.  Despite prior care taken to avoid doing this, I still managed to commit it in the end.  Now fixed.

**Pull request merged**